Device/_Template_Flash.deprecated/Abstract.txt

```diff
@@ -0,0 +1,32 @@
+Creating a new Algorithm
```

Comment:
Do we need to keep this deprecated version? Is it still relevant? Or could we simply drop it from the head? One can always grab the old version from the history.

Reply:
I would suggest to keep this version for consistency reasons.